mimecast/assets/dashboards/mimecast_ttp_url.json

```diff
@@ -0,0 +1,748 @@
+{
+    "title": "Mimecast - TTP URL",
+    "description": "This log shows the details of events which were generated and flagged by TTP URL configuration because the user clicks on a malicious link.",
+    "widgets": [
+        {
+            "id": 4947291615075058,
+            "definition": {
+                "type": "image",
+                "url": "https://primasecure.com/wp-content/uploads/2021/07/Mimecast_M_2015.png",
+                "sizing": "contain",
+                "margin": "md",
+                "has_background": false,
+                "has_border": true,
+                "vertical_align": "center",
+                "horizontal_align": "center"
+            },
+            "layout": {
+                "x": 0,
+                "y": 0,
+                "width": 4,
+                "height": 2
+            }
+        },
+        {
+            "id": 1004363650832560,
+            "definition": {
+                "type": "note",
+                "content": "## Overview\n\nThis log shows the details of events which were generated and flagged by TTP URL configuration because the user clicks on a malicious link.",
```

Comment:
```suggestion
                "content": "## Overview\n\nThis log shows the details of events which were generated and flagged by TTP URL configuration because the user clicked on a malicious link.",
```